sdk/openai/azure-ai-openai-assistants/src/test/java/com/azure/ai/openai/assistants/AzureRetrievalAsyncTest.java

```diff
@@ -0,0 +1,122 @@
+package com.azure.ai.openai.assistants;
```

Comment:
missing 

// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.